<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OivObjectenDockWidgetBase</class>
 <widget class="QDockWidget" name="OivObjectenDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>505</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>Importeer bestand</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="bouwlaag_id">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="object_id">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label1_2">
        <property name="text">
         <string>Alles wat u importeert wordt gekoppeld aan:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="object">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(208, 208, 208);
border-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>1. Selecteer het Shape of DXF bestand welke geimporteerd moet worden.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="select_file">
        <property name="text">
         <string>Selecteer bestand</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bestandsnaam">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(208, 208, 208);
border-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label3">
        <property name="text">
         <string>2. Selecteer het veld welke de waarde voor het type bevat:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="type"/>
      </item>
      <item>
       <widget class="QLabel" name="label6">
        <property name="text">
         <string>4. Maak een mapping aan voor de types:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mapping">
        <property name="text">
         <string>Maak mapping</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label7">
        <property name="text">
         <string>5. Alle controles zijn goed u kunt importeren.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="import_file">
        <property name="text">
         <string>Importeer</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="terug">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Terug</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
